.gitignore

```diff
@@ -1,3 +1,3 @@
 #dependencies
-/node_modules
-text.md
\ No newline at end of file
+node_modules
+test.md
```

Comment:
- [ ] You have added `test.md` file in the `gitignore` but your project still has a `test.md` file. Kindly remove the `test.md` file from your project. :+1: